Dockerfile.test

```diff
@@ -17,10 +17,10 @@ COPY . /fauna/faunadb-python
 
 # Shift over to the project and install the dependencies
 WORKDIR /fauna/faunadb-python
-RUN pip install .
+RUN pip install . .[test]
```

Comment:
That's a weird looking argument...

Reply:
It means to install the test group of dependencies for the current folder